packages/editor/src/store/selectors.js

```diff
@@ -754,6 +766,11 @@ export const getGlobalBlockCount = createSelector(
 	]
 );
 
+const mapClientIds = ( clientIds, fn ) => map(
```

Comment:
Not loving a randomly-placed, un-documented, un-tested utility function located here. Thinking either placement in a utilities file, or placement at top of function to have some arrangement of selectors vs. non-selectors. Is it just handling the coercion of `castArray`?

I don't see a ton of value in pulling this out as a function vs. just inlining.

Reply:
Removed, no longer needed.